### PR TITLE
enable gstreamer 1.x

### PR DESCRIPTION
**Pull request by foolab**
please review but do not merge. I will do the final merge when everything is in OBS.

### CODE REVIEW

File: rpm/xulrunner-qt5.spec [hunk 1]
```diff
@@ -126,6 +126,7 @@ echo "ac_add_options --disable-javaxpcom" >> mozconfig
 echo "ac_add_options --disable-crashreporter" >> mozconfig
 echo "ac_add_options --without-x" >> mozconfig
 echo "ac_add_options --with-app-name=%{name}" >> mozconfig
+echo "ac_add_options --enable-gstreamer=1.0" >> mozconfig
```
Comment (rainemak) [inline rpm/xulrunner-qt5.spec]: Worth adding `ac_add_options --enable-gstreamer=1.0` directly to the `embedding/embedlite/config/mozconfig.merqtxulrunner`

**Comment by tworaz** [comment]
LGTM

**Comment by rainemak** [comment]
LGTM

**Comment by rainemak** [comment]
Please push the same PR to https://github.com/tmeshkova/gecko-dev embedlite_esr branch.

**Comment by rojkov** [comment]
Tested with latest xulrunner and gstreamer1.0. No more twisted video.

**Pull request merged**